src/sentry/api/serializers/models/projectcodeowners.py

```diff
@@ -0,0 +1,17 @@
+from sentry.api.serializers import Serializer, register
+from sentry.models import ProjectCodeOwners
+
+
+@register(ProjectCodeOwners)
+class ProjectCodeOwnersSerializer(Serializer):
+    def serialize(self, obj, attrs, user):
+        data = {
+            "raw": obj.raw,
+            "dateCreated": obj.date_added,
+            "dateUpdated": obj.date_updated,
+            "provider": "unknown",
+        }
+        if obj.organization_integration:
+            data["provider"] = obj.organization_integration.integration.provider
+
+        return data
```

Comment:
I haven't added the code mapping in here, not sure if we just want the id or more information

Reply:
@MeredithAnya hmm I think it would be good to populate the associated code path mapping

Reply:
okay I'll add the `id` for now